src/components/expand-viewer/index.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FC, useState } from 'react';
+import Popup, { PopupProps } from '../popup';
+import IconButton from '../icon-button';
+import { getIcon, IconName } from '../icon/list';
+import { useTheme } from 'emotion-theming';
+import { Box, Flex } from 'rebass';
+import Value from '../typography/value';
+import Subtitle from '../typography/subtitle';
+import { expandViewButtonStyle } from './expand-viewer.styles';
+import { ITheme } from '../../theme/types';
+
+// ExpandViewer
+export interface ExpandViewerProps {
+  title?: string;
+  NormalComponent: FC;
+  BriefComponent: FC;
+  popupProps?: Omit<PopupProps, 'children' | 'isOpen' | 'onClose'>;
+}
+
+const ExpandViewer: FC<ExpandViewerProps> = ({
+  title,
+  NormalComponent,
+  BriefComponent,
+  popupProps,
+}) => {
+  const [expanded, setExpanded] = useState(false);
```

Comment:
there is `usePopup`, which is very suitable for cases like this.
Have you considered using it?

Reply:
yeah checked that but it's not suitable for this case since the second returned is the handleToggle which we don't need here.